xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-ui/src/main/resources/XWiki/DeletedDocuments.xml

```diff
@@ -43,7 +43,7 @@ $xwiki.jsx.use('XWiki.DeletedDocuments')##
 ##
 #set($columns = ["ddoc.fullName", "ddoc.title", "ddoc.date", "ddoc.deleter", 'ddoc.batchId', 'actions'])
 #set($columnsProperties = {
-    'ddoc.fullName'  : { 'type' : 'text', 'size' : 10 },
+    'ddoc.fullName'  : { 'type' : 'page', 'size' : 10 },
```

Comment:
I'm not sure the new page filter is better (in this case) than the previous text filter. The suggest-based live table filter works best IMO when:

* the user is interested mainly in exact match (you can still do partial matching, but the main use case is exact matching)
* the value you select from the suggest picker appears on multiple rows, i.e. the value is shared by multiple entities listed in the live table, i.e. this value is not the main information from the live table

For instance, if the live table displays the users, it doesn't make sense to have a suggest-based filter for the first column (user name) because the result of the filtering will be zero or one row. On the other hand, if the list table displays pages, then it makes sense to have a suggest-based filter for the page author.

In the case of the page filter, it would make sense if the live table displays entities that have a relation n-to-1 with pages. A live table that displays objects (e.g. comments) or attachments for instance (but even there it's debatable if you want partial matching of exact matching).

Reply:
@evalica WDYT? I don't think it brings much value to use the page autocomplete / suggest on the Deleted Pages / Attachments index. I think the free text filtering is more suited there. Do you agree?

@atallahade I agree that without a real use case it doesn't make sense to implement the live table page picker (it won't be tested and thus it could easily get broken without anyone noticing it).

Reply:
First of all I don't think it's nice that Main.AllDocs 'Index' tab is not consistent with 'Deleted Pages'. Index is focused on page titles (as it should, since it's an application targeted towards normal users), while 'Deleted Pages' shows page names. 

I consider that Page filter is a 'component' that should be used for technical users, where it's vital that we specify the 'page name' (usually when referencing class names, etc.) and we cannot use 'page title' instead. I don't believe "Deleted Pages" to be this case. 

Also 'page filter' should be used in cases we don't have filtering present in other ways. In the livetable case, we are duplicating the functionality. The user will see first the result in the 'page filter' popover, then the result will be applied / refreshed in the livetable. For me, this is like an interaction that instead of taking 2 seconds, it's doubling, taking it 4 seconds to receive the same results. 

I don't think adding the 'page filter' in the 'Deleted Pages' will bring any improvements to the normal user. Also I think we should use this filter just for tech / development / advanced user use cases. 

What I would want to see on the 'Deleted Pages' is: 
* Instead of 'Page' and 'Title' columns, to have 'Title' and 'Location' columns instead (consistent with Index tab)
* That the user could filter the 'Title' column, which is not possible now
* We should preserve the 'text' filter on the livetable

We could test the following use case: the user accidentally deleted the test pages from Sandbox. He goes to the Recycle Bin and wants to restore the pages. He knows the pages have in their title something that contains the "test page" words. Help him find those pages :)

Reply:
'Deleted Attachments' should also be made consistent with 'Attachments': have 'Location' column instead of 'Page' (name.

Reply:
I guess the best thing to do is to create dedicated jira issues for what you are describing. I'll close this PR and mark the current issue as "won't do".

Reply:
Created https://jira.xwiki.org/browse/XWIKI-15813